src/Axe.Windows.TelemetryTests/Axe.Windows.TelemetryTests.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props')" />
```

Comment:
>1.3.2 [](start = 50, length = 5)

Please use version 1.4.0 of the test adapter (I just updated these recently to deal with some weird test behavior in Visual Studio)